freqtrade/optimize/__init__.py

```diff
@@ -12,6 +11,11 @@
 from freqtrade.exchange import Exchange
 from freqtrade.arguments import TimeRange
 
+import importlib
+ujson_found = importlib.util.find_spec("ujson")
+if ujson_found is not None:
+    import ujson
```

Comment:
if you use `import ujson as json` it will act as a drop-in replacement ... so you don't need to check below

Reply:
+ add a debug-logentry showing this was used ...

Reply:
tested this option and works great just replaced:
`import json` with `import ujson as json`

Reply:
@macd2 you may want to DL the latest commit. 
FP is handled differently to python json without a kwarg now added.

Reply:
As already mentioned by @xmatthias the check in `trim_tickerlist` should be removed and handled via imports, for example:
```
try:
    import ujson as json
except ImportError:
    import json
```